doc/src/sgml/ddl.sgml

```diff
@@ -4594,9 +4616,10 @@ REVOKE CREATE ON SCHEMA public FROM PUBLIC;
 -->
 一般ユーザに、ユーザ個人用のスキーマだけを使わせます。
 これを実現するには、<literal>REVOKE CREATE ON SCHEMA public FROM PUBLIC</literal>を発行し、個々のユーザにユーザと同じ名前でスキーマを作成してください。
-以前からログインしているユーザに対しては、<literal>pg_catalog</literal>スキーマ内にあるのと同じ名前のpublicスキーマ内のオブジェクトを監視することを考慮してください。
 デフォルトサーチパスは、ユーザ名として解釈される<literal>$user</literal>で始まることを思い出してください。
-ですから、デフォルトではユーザは別々のスキーマを所有し、自身のスキーマにアクセスします。
+ですから、ユーザが各自別々のスキーマを所有しているなら、デフォルトでは自身のスキーマにアクセスします。
+信頼できないユーザがすでにログインしたデータベースでこのパターンを採用した後には、<literal>pg_catalog</literal>スキーマ内にあるのと同じ名前のpublicスキーマ内のオブジェクトを監視することを考慮してください。
+セキュアなスキーマ利用パターンが存在しないような、信用できないユーザがデータベース所有者である場合や、<literal>CREATEROLE</literal>権限を持っている場合を除き、このパターンはセキュアなスキーマ利用パターンです。
       </para>
      </listitem>
 
```

Comment:
(diffだと分かりにくいですが)以下のブロックにある変更が反映されていないようです。

Reply:
「以下のブロック」とはどこでしょうか？

Reply:
以下の箇所です。
<!--
       Remove the public schema from the default search path, by modifying
       <link linkend="config-setting-configuration-file"><filename>postgresql.conf</filename></link>
       or by issuing <literal>ALTER ROLE ALL SET search_path =
       "$user"</literal>.  Everyone retains the ability to create objects in
       the public schema, but only qualified names will choose those objects.
       While qualified table references are fine, calls to functions in the
       public schema <link linkend="typeconv-func">will be unsafe or
       unreliable</link>.  If you create functions or extensions in the public
       schema, use the first pattern instead.  Otherwise, like the first
       pattern, this is secure unless an untrusted user is the database owner
       or holds the <literal>CREATEROLE</literal> privilege.
-->
「by issuing <literal>ALTER ROLE ALL SET search_path =   "$user"</literal>.」が加えられていたり、や
Everyone retains the ability〜以下の訳文が合っていないように思います。

Reply:
なるほど、ありがとうございます。修正しました。